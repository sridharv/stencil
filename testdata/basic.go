package basic

func Max(a, b int) int {
	if a > b {
		return a
	}
	return b
}
