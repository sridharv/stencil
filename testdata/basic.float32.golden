package basic

func Max(a, b float32) float32 {
	if a > b {
		return a
	}
	return b
}
